UT4MasterServer/Controllers/UnrealTournamentStatsController.cs

```diff
@@ -15,68 +17,71 @@ namespace UT4MasterServer.Controllers;
 public class UnrealTournamentStatsController : JsonAPIController
 {
 	private readonly ILogger<UnrealTournamentStatsController> logger;
+	private readonly StatisticsService statisticsService;
 
-	public UnrealTournamentStatsController(ILogger<UnrealTournamentStatsController> logger)
+	public UnrealTournamentStatsController(ILogger<UnrealTournamentStatsController> logger, StatisticsService statisticsService)
 	{
 		this.logger = logger;
+		this.statisticsService = statisticsService;
 	}
 
-	[HttpGet("accountId/{id}/bulk/window/{category}")]
-	public IActionResult Stats(string id, string leagueName, string category)
+	// Examples:
+	// /ut/api/stats/accountId/0b0f09b400854b9b98932dd9e5abe7c5/bulk/window/daily
+	[HttpGet("accountId/{id}/bulk/window/daily")]
+	public async Task<IActionResult> GetDailyAccountStatistics(string id)
 	{
-		// TODO: Check id and return ErrorResponse if invalid: "Could not convert fakeAccountId to a UUID because it was not in a valid format: Invalid UUID string: fakeAccountId" (errors.com.epicgames.modules.stats.invalid_account_id)
-
-		// TODO: return Stats
+		var accountId = EpicID.FromString(id);
```

Comment:
Leave comment that you didn't handle
```suggestion
	// TODO: Check id and return ErrorResponse if invalid: "Could not convert fakeAccountId to a UUID because it was not in a valid format: Invalid UUID string: fakeAccountId" (errors.com.epicgames.modules.stats.invalid_account_id)
	var accountId = EpicID.FromString(id);
```